data/ca/retired/Amy-Bublak-9a0870f9-bbea-4703-98af-7d307df94060.yml

```diff
@@ -0,0 +1,13 @@
+id: ocd-person/9a0870f9-bbea-4703-98af-7d307df94060
+name: Amy Bublak
+given_name: Amy
+family_name: Bublak
+email: abublak@turlock.ca.us
+roles:
+- end_date: '2022-12-31'
```

Comment:
@bfossen-ce this is a duplicate from earlier in the pull. This person was also re-elected and is not retiring